sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_v1/schema.yaml

```diff
@@ -0,0 +1,31 @@
+fields:
+- mode: NULLABLE
+  name: date
+  type: DATE
+- mode: NULLABLE
+  name: country_name
+  type: STRING
+- mode: NULLABLE
+  name: channel
+  type: STRING
+- mode: NULLABLE
+  name: build_id
+  type: STRING
+- mode: NULLABLE
+  name: os
+  type: STRING
+- mode: NULLABLE
+  name: os_version
+  type: NUMERIC
```

Comment:
I think it will be surprising to have this field stored as `NUMERIC` and that we should cast it to string.